english/eating-walls/find-a-toilet/relieve/relieve-yourself.md

```diff
@@ -1,3 +1,7 @@
 Gross!
 
 You turn and run away in shame.
+
+But almost immediately you slam straight into a pole and break your front teeth.  You cry out in agony and turn toward a faint light coming from a door you hadn't seen before.  When you reach the door, you see it is a dental office.  How convenient.
+
+[You go inside](bite-the-marshmallow/dentist/dentist-office.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.